packages/repository/README.md

```diff
@@ -21,47 +21,137 @@ $ npm install --save @loopback/repository
 
 ## Basic use
 
+At the moment, we only have implementations of `Repository` based on LoopBack
+3.x `loopback-datasource-juggler` and connectors. The following steps illustrate
+how to define repositories and use them with controllers.
+
+### Defining a legacy datasource and a model
+
 The repository module provides APIs to define LoopBack 3.x data sources and
 models. For example,
 
 ```ts
-import {
-  DataSourceConstructor,
-  juggler,
-  Entity,
-  model,
-  ModelDefinition
-} from '@loopback/repository';
+// src/datasources/db.datasource.ts
+import {juggler, DataSourceConstructor} from '@loopback/repository';
 
-export const ds: juggler.DataSource = new DataSourceConstructor({
+export const db: juggler.DataSource = new DataSourceConstructor({
```

Comment:
Not sure why switch to `db`. Please note `ds` stands for `dataSource`.

Reply:
I switched the constant name to `db` because when we import the datasource to be registered with `app.datasource()`, we retrieve it back through dependency injection via the string: `datasources.db` and NOT `datasources.ds`, the name of the constant. I thought we should be consistent here as to not confuse the users.

Alternatively, I could change `name: 'db'` to `name: 'ds'` so that the name of the variable being named is consistent with the datasource's name. Should I switch back regardless?

Reply:
+1 for using `db`, it's the name of the default we use in our project templates (LB 3.x and 4.x) too.

Not a big deal though, I can live with `ds` too.